mkosi/__init__.py

```diff
@@ -2241,6 +2242,95 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    try:
+        from _emerge.actions import action_build, action_sync, load_emerge_config
+
+        if "build-script" in args.debug:
+            from _emerge.actions import action_info
+    except ImportError:
+        die("You need portage module for Gentoo: https://gitweb.gentoo.org/proj/portage.git")
+
+    import multiprocessing
```

Comment:
Move this one to the top of the file with the rest of the imports

Reply:
done